tests/playwright/src/utility/operations.ts

```diff
@@ -181,27 +183,57 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
   const podmanResourceCard = new ResourceConnectionCardPage(page, RESOURCE_NAME, machineVisibleName);
   await playExpect(podmanResourceCard.providerConnections).toBeVisible({ timeout: 10_000 });
   await waitUntil(
-    async () => {
+    async function machineExists() {
       return await podmanResourceCard.resourceElement.isVisible();
     },
-    { timeout: 15000 },
+    { timeout: 15000, sendError: false },
   );
   if (await podmanResourceCard.resourceElement.isVisible()) {
     await playExpect(podmanResourceCard.resourceElementConnectionActions).toBeVisible({ timeout: 3000 });
     await playExpect(podmanResourceCard.resourceElementConnectionStatus).toBeVisible({ timeout: 3000 });
+    if ((await podmanResourceCard.resourceElementConnectionStatus.innerText()) === ResourceElementState.Starting) {
+      console.log('Podman machine is in starting currently, will send stop command via CLI');
+      execSync(`podman machine stop ${machineVisibleName}`);
+      await playExpect(podmanResourceCard.resourceElementConnectionStatus).toHaveText(ResourceElementState.Off, {
+        timeout: 30_000,
+      });
+      console.log('Podman machine stopped via CLI');
+    }
     if ((await podmanResourceCard.resourceElementConnectionStatus.innerText()) === ResourceElementState.Running) {
       await podmanResourceCard.performConnectionAction(ResourceElementActions.Stop);
       await playExpect(podmanResourceCard.resourceElementConnectionStatus).toHaveText(ResourceElementState.Off, {
-        timeout: 30_000,
+        timeout: 300_000,
```

Comment:
300 seconds for a machine to go offline seems a bit excessive, I'd say if it takes that long, that's a bug and should be reported to devs